addons/website_livechat/tests/test_chatbot_ui.py

```diff
@@ -139,3 +140,18 @@ def test_chatbot_redirect(self):
         default_website.channel_id = livechat_channel.id
         self.env.ref("website.default_website").channel_id = livechat_channel.id
         self.start_tour("/contactus", "website_livechat.chatbot_redirect")
+
+    def test_chatbot_fw_operator_matching_lang(self):
+        fr_op = self._create_operator(lang_code="fr_FR")
+        en_op = self._create_operator(lang_code="en_US")
+        self.livechat_channel.user_ids = fr_op + en_op
+        self.env["discuss.channel"].search([("livechat_channel_id", "=", self.livechat_channel.id)]).unlink()
+        self.start_tour("/", "website_livechat.chatbot_fw_operator_matching_lang", cookies={"frontend_lang": "fr_FR"})
+        channel = self.livechat_channel.channel_ids[0]
+        self.assertIn(channel.channel_member_ids.partner_id.user_ids, fr_op)
+        self.assertNotIn(channel.channel_member_ids.partner_id.user_ids, en_op)
```

Comment:
Can't we assert equal instead of checking each option separately?

Reply:
I don't think so? In this case the live chat operator will be the chat bot, so we need to check whether the operator was added to the members. And while we're at it, better also ensure the other one is not in the members?